packages/web-components/@sfx/ui/CHANGELOG.md

```diff
@@ -0,0 +1,9 @@
+# Changelog
+All notable changes to this project will be documented in this file.
+
+The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
+and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
+
+## [Unreleased]
+### Added
+- SFX-152: Added the list sub-component.
```

Comment:
I would drop the "sub-" because the list component does not rely on any larger component to be usable.